### PR TITLE
katello_host_tools_host no disable sca

### PR DESCRIPTION
**Pull request by pondrejk**
### Problem Statement
sca default in 6.16

### Solution
rid of sca_disable

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py -k test_positive_report_package_installed_removed[rhel7]

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6674
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_positive_report_package_installed_removed[rhel7] --external-logging
Test Result : =========== 89 deselected, 98 warnings, 1 error in 719.67s (0:11:59) ===========
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py -k test_positive_report_package_installed_removed[rhel7]

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6678
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_positive_report_package_installed_removed[rhel7] --external-logging
Test Result : ========== 1 failed, 89 deselected, 98 warnings in 751.33s (0:12:31) ===========
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py -k test_positive_install_package_via_rex[rhel7]

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6815
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_positive_install_package_via_rex[rhel7] --external-logging
Test Result : ========== 1 failed, 85 deselected, 98 warnings in 1051.08s (0:17:31) ==========
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py -k  test_positive_package_applicability[rhel8]

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6831
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py -k  test_positive_package_applicability[rhel8] --external-logging
Test Result : ========== 1 failed, 85 deselected, 94 warnings in 758.76s (0:12:38) ===========
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py  -k test_syspurpose

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6858
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py  -k test_syspurpose --external-logging
Test Result : ========== 1 failed, 86 deselected, 96 warnings in 423.23s (0:07:03) ===========
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6865
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_syspurpose --external-logging
Test Result : ========== 1 failed, 86 deselected, 96 warnings in 733.22s (0:12:13) ===========
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py -k test_syspurpose

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6869
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_syspurpose --external-logging
Test Result : ========== 1 failed, 86 deselected, 96 warnings in 748.99s (0:12:28) ===========
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py -k test_syspurpose

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6870
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_syspurpose --external-logging
Test Result : ========== 1 failed, 86 deselected, 96 warnings in 714.38s (0:11:54) ===========
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py -k test_positive_package_applicability[rhel8]

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6877
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_positive_package_applicability[rhel8] --external-logging
Test Result : ========== 1 passed, 86 deselected, 95 warnings in 744.92s (0:12:24) ===========
```

**Comment by pondrejk** [comment]
sorry for the excess of prt comments (local setup issues on my side)

**Review by vsedmik — APPROVED** [review]

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_host.py -k test_syspurpose

File: tests/foreman/cli/test_host.py [hunk 1]
```diff
@@ -2294,7 +2269,7 @@ def test_syspurpose_end_to_end(
     )
     host = target_sat.cli.Host.info({'id': host['id']})
     # Assert system purpose values have been updated in the host as expected
-    assert host['subscription-information']['system-purpose']['purpose-addons'] == "test-addon3"
+    assert host['subscription-information']['system-purpose']['purpose-addons'][0] == "test-addon3"
     assert host['subscription-information']['system-purpose']['purpose-role'] == "test-role2"
     assert host['subscription-information']['system-purpose']['purpose-usage'] == "test-usage2"
     assert host['subscription-information']['system-purpose']['service-level'] == "Self-Support2"
```
Comment (vsedmik) [inline tests/foreman/cli/test_host.py:2275]: Probably the following will lines need a change too
```
host_subscriptions = target_sat.cli.ActivationKey.subscriptions(
        {
            'organization-id': module_org.id,
            'id': activation_key.id,
            'host-id': host['id'],
        },
        output_format='json',
    )
assert len(host_subscriptions) > 0
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7010
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_syspurpose --external-logging
Test Result : ========== 1 failed, 86 deselected, 95 warnings in 719.75s (0:11:59) ===========
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7013
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_host.py -k test_syspurpose --external-logging
Test Result : ========== 1 passed, 86 deselected, 95 warnings in 819.98s (0:13:39) ===========
```

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**